### DISCUSSION TITLE
Case-based reasoning

### DISCUSSION THREAD

**Discussion by VitaliyMez** [denkbares/d3web-KnowWE#20] [Ideas]
Hi,

what is markup to support Case-based reasoning in KnowWE?
As far as I see, TestCases can be used used for the proof of general knowledge? 
If there is a possibility for generalization and further deduction from TestCases? 

Thank you!

**Comment by AlbrechtStriffler** [comment]
Hi Vitaliy,

KnowWE doesn't have markup support for case-based reasoning at the moment, at least as far as I know. You might want to check out Covering Lists though (or XCL for eXtended Covering Lists as we call it in KnowWE), it has a similar approach.

- Docu page: https://www.d3web.de/Wiki.jsp?page=Doc%20CoveringList
- Small example (bottom of page): https://www.d3web.de/Wiki.jsp?page=Demo%20-%20FlatBattery

Best Regards
Albrecht
